src/static-pages/styles/author-guide.scss

```diff
@@ -0,0 +1,40 @@
+.author-guide-page {
+    grid-area: static-nav;
+    display: flex;
+    flex-direction: column;
```

Comment:
could this be a single column grid instead of flex?

Reply:
Can be addressed as another ticket since there is one for the left hand menu.

Reply:
ok